modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {documentLibraryPagesTest} from './fixtures/documentLibraryPagesTest';
+
+export const test = mergeTests(
```

Comment:
This `export` isn't needed 😕 we are not importing it anywhere

Reply:
I am checking that export and it looks like it is used by the test itself.

In fact, as far as I can see all the other tests have their associated export test. So it looks like it is being used.

Reply:
Now I see, I thought you were referring to the entire mergeTests block. I have already removed the export.

Thanks for the advice.

Fixed.